pkg/utils/partitions/getpartitions.go

```diff
@@ -209,3 +209,24 @@ func GetPartitionViaDM(fs v1.FS, label string) *v1.Partition {
 	}
 	return part
 }
+
+func IsPathMounted(fs v1.FS, s string) (bool, error) {
```

Comment:
dont we have a util from the mount lib that checks this? Maybe we dont need to reimplement it already?

Reply:
yep, there is a `IsNotMountPoint` from the mounter, maybe we can reuse it?

```
// IsNotMountPoint determines if a directory is a mountpoint.
// It should return ErrNotExist when the directory does not exist.
// IsNotMountPoint is more expensive than IsLikelyNotMountPoint.
// IsNotMountPoint detects bind mounts in linux.
// IsNotMountPoint enumerates all the mountpoints using List() and
// the list of mountpoints may be large, then it uses
// isMountPointMatch to evaluate whether the directory is a mountpoint.
func IsNotMountPoint(mounter Interface, file string) (bool, error) {
```

Reply:
You also got `i.cfg.Mounter.IsLikelyNotMountPoint(constants.UkiCdromSource)` available but we are using a fakeMounter so we need to set a proper mounter on config build for that to work.

Whatever you wish, but thats the mounter check from kubernetes so I would say that has been thoroughly tested lol

Reply:
> You also got `i.cfg.Mounter.IsLikelyNotMountPoint(constants.UkiCdromSource)` available but we are using a fakeMounter so we need to set a proper mounter on config build for that to work.
> 
> Whatever you wish, but thats the mounter check from kubernetes so I would say that has been thoroughly tested lol

alright alright, moving to using IsMountPoint() then :)